python/pyspark/sql/catalog.py

```diff
@@ -37,11 +37,19 @@ class Database(NamedTuple):
 
 class Table(NamedTuple):
```

Comment:
We will have to document this at https://github.com/apache/spark/tree/master/python/docs/source/reference but it's not there already. Let's do it later separately.